double-conversion/strtod.cc

```diff
@@ -202,6 +202,10 @@ static bool DoubleStrtod(Vector<const char> trimmed,
                          int exponent,
                          double* result) {
 #if !defined(DOUBLE_CONVERSION_CORRECT_DOUBLE_OPERATIONS)
+  // Avoid "unused parameter" warnings
+  (void)trimmed;
+  (void)exponent;
+  (void)result;
```

Comment:
Nit, but the other locations in the code use a space.
```suggestion
  (void) trimmed;
  (void) exponent;
  (void) result;
```

Reply:
Done.